### ISSUE TITLE
updating to version 4.7.1 has broken Matomo

### ISSUE THREAD

**Issue by mkassir95** [piwik/piwik#18745]
updating to version 4.7.1 has broken Matomo .
I was using version 4.6.2
I used automatic update option 
I restarted my server and the problem persists

![image](https://user-images.githubusercontent.com/72728047/152571629-b0e387d1-9b68-4f62-a7b2-ecf777cf14a3.png)

**Comment by Findus23** [comment]
Can you please check the PHP or Matomo error log for more details on what exactly doesn't work?

**Comment by rr0001** [comment]
I also had the same error, plus another that said:

```
A fatal error occurred
The following error just broke Matomo (v4.7.1):

An exception has been thrown during the rendering of a template ("File `../../CoreHome/vue/src/MenuDropdown/MenuDropdown.less` not found. in anonymous-file-0.less").
in C:\Projects\opt\www\localhost\matomo\plugins\CoreUpdater\templates\runUpdaterAndExit_welcome.twig line 8            
Troubleshooting
Follow these steps to solve the issue or report it to the team:
If you have just updated Matomo to the latest version, please try to restart your web server. This will clear the PHP opcache which may solve the problem.
If this is the first time you see this error, please try refresh the page.
If this error continues to happen, we appreciate if you send the [error report](mailto:hello@matomo.org?subject=Fatal%20error%20in%20Matomo%204.7.1&body=An%20exception%20has%20been%20thrown%20during%20the%20rendering%20of%20a%20template%20%28%22File%20%60..%2F..%2FCoreHome%2Fvue%2Fsrc%2FMenuDropdown%2FMenuDropdown.less%60%20not%20found.%20in%20anonymous-file-0.less%22%29.%20in%20C%3A%5CProjects%5Copt%5Cwww%5Clocalhost%5Cmatomo%5Cplugins%5CCoreUpdater%5Ctemplates%5CrunUpdaterAndExit_welcome.twig%208%20using%20PHP%208.0.12) to the Matomo team.
```

I was able to resolve the issue by manually downloading the zip file and overwriting all the files. I also had backed up my `config/config.inc.php` file.

**Comment by mkassir95** [comment]
Yes this error also appeared in my case but i was not able to reproduce it

**Comment by pietryszak** [comment]
Same issue, after update from 4.6.2 to 4.7.1 and server restart the same info is shown

**Comment by sgiehl** [comment]
@pietryszak What file system are you using on your server? Is it case sensitive or case insensitive?

**Comment by pietryszak** [comment]
> 

Ubuntu 20.04.4 LTS, ext4. With default file system settings. 
I restore server from backup and old version of Matomo works fine.

**Comment by sgiehl** [comment]
Closing this one, as it should be finally solved with #19156
